templates/server-statefulset.yaml

```diff
@@ -169,7 +169,7 @@ spec:
           livenessProbe:
             httpGet:
               path: {{ .Values.server.livenessProbe.path | quote }}
-              port: 8200
+              port: .Values.server.livenessProbe.port
```

Comment:
```suggestion
              port: {{ .Values.server.livenessProbe.port }}
```

Reply:
Silly mistakes. :(

Reply:
Actually, in case of more than one tcp listener using service's or server's listening port will not help.